lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2937,46 +2937,57 @@ namespace {
       if (!clangDecl->hasAttrs())
         return;
 
-      SmallVector<ValueDecl *, 1> results;
-      auto conformsToAttr =
-          llvm::find_if(clangDecl->getAttrs(), [](auto *attr) {
-            if (auto swiftAttr = dyn_cast<clang::SwiftAttrAttr>(attr))
-              return swiftAttr->getAttribute().starts_with("conforms_to:");
-            return false;
-          });
-      if (conformsToAttr == clangDecl->getAttrs().end())
+      SmallVector<clang::Attr*, 40> conformsToMatches;
```

Comment:
I am not sure what number of conformances do we expect for a class, but 40 sounds a bit too much for me. Note that if we go over that only results in a heap allocation, so this is strictly a performance optimization not a correctness issue if the user has more.

Reply:
Agreed. Do we have data regarding the median amount of conformances?

Reply:
I think using `1` as a number of conformances here would be reasonable. There wouldn't usually be many conformances.